omnibus/config/software/dbus.rb

```diff
@@ -0,0 +1,44 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2016-present Datadog, Inc.
+
+name "dbus"
+default_version "1.15.4"
+
+dependency "expat"
+dependency "libtool"
+dependency "pkg-config"
+
+license "AFL-2.1"
+license "GPL-2.0-or-later"
+license_file "LICENSES/AFL-2.1.txt"
+license_file "LICENSES/GPL-2.0-or-later.txt"
```

Comment:
The https://gitlab.freedesktop.org/dbus/dbus/-/blob/master/COPYING file says:

```
dbus is licensed to you under your choice of the Academic Free
License version 2.1, or the GNU General Public License version 2
(or, at your option any later version).

<snip>

Some of the standalone binaries are under the GPL only; in particular,
but not limited to, tools/dbus-cleanup-sockets.c and test/decode-gcov.c.
```

If we don't ship these binaries, I think we can just choose which license we want and we don't need to list them both?

Reply:
I've removed the dbus tools and chose the more permissive AFL license.